lightkurve/targetpixelfile.py

```diff
@@ -833,84 +893,6 @@ def get_bkg_lightcurve(self, aperture_mask=None):
 
-
-    def _get_hdu(image, extension):
-        """Checks if an image is a single FITS image, or a FITS file,
-        and returns the appropriate PrimaryHDU.
-
-        Attributes
-        ----------
-        image : fits.ImageHDU or fits.HDUList object
-            FITS filename path or ImageHDU object to get the data from.
-        extension: integer
-            Number of the file extension containing data, depends on the file
-            type and source.
-
-        Returns
-        -------
-        hdu : astropy.PrimaryHDU
-            PrimaryHDU of the image inputted.
-        """
-        if isinstance(image, fits.ImageHDU):
-            hdu = image
-        elif isinstance(image, fits.HDUList):
-            hdu = image[extension]
-        else:
-            hdu = fits.open(image)[extension]
-        return hdu
-
-    def _get_pos_corrs(mid_img, hdu, position, extension):
-        """Calculates the position correction values (both in row and column) and
-        WCS for an image relative to the reference image, defined as the middle
-        cadence of the FITS file.
-
-        Attributes
-        ----------
-        hdu : astropy.PrimaryHDU
-            PrimaryHDU of the image being looked at.
-        mid_img : fits.ImageHDU
-            Reference image from the FITS file.
-        position : astropy.SkyCoord
-            Position around which to cut out pixels.
-        extension : integer
-            Number of the file extension containing data, depends on the file
-            type and source.
-
-        Returns
-        -------
-        poscorr1 :  float32
-            Position correction in the vertical direction (column).
-        poscorr2 :  float32
-            Position correction in the horizontal direction (row).
-        column : integer
-            Column number of the center pixel.
-        row : integer
-            Row number of the center pixel.
-        ractr :
-            Right ascension of the center of the center pixel.
-        decctr:
-            Declination of the center of the center pixel.
-        """
-        # Calculate reference WCS coords from the middle image to use later to
-        # do pos_corr calcs. Get the middle pixel column and row.
-        mid_hdu = KeplerTargetPixelFile._get_hdu(mid_img, extension)
-        wcs = WCS(mid_hdu)
-        pixelpos = skycoord_to_pixel(position, wcs=wcs)
-        column, row = int(np.round(pixelpos[0])), int(np.round(pixelpos[1]))
-
-        # Calculate the ra and dec of the center pixel of the middle image
-        skyposctr = pixel_to_skycoord(column, row, wcs=wcs)
-        ractr = skyposctr.ra.deg
-        decctr = skyposctr.dec.deg
-
-        # Get positional shift of the image compared to the reference file
-        wcs1 = WCS(hdu)
-        pixelpos1 = skycoord_to_pixel(position, wcs=wcs1)
-        poscorr1 = pixelpos1[0] - pixelpos[0]
-        poscorr2 = pixelpos1[1] - pixelpos[1]
-
-        return [poscorr1, poscorr2, column, row, ractr, decctr]
```

Comment:
Two questions/issues here: 
Where in the code are ractr and decctr actually used? I am not seeing this. And if they are the coordinates of the center of the middle fits file, then there is no  need to have them in this loop, because they are being calculated over and over again.